src/x3d-parser.js

```diff
@@ -545,19 +545,24 @@ function renderX3D(THREE, x3dXml, scene, useImageTexture) {
 
         var tree = { 'string': 'Scene', children: [] };
 
-        parseChildren(x3dXml.documentElement.childNodes[1], tree);
```

Comment:
The `Scene` tag isn't necessarily at index 1. For example, the following is Blender-exported X3D. It's better to validate that we're actually finding the Scene before beginning traversing the Scene

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE X3D PUBLIC "ISO//Web3D//DTD X3D 3.0//EN" "http://www.web3d.org/specifications/x3d-3.0.dtd">
<X3D version="3.0" profile="Immersive" xmlns:xsd="http://www.w3.org/2001/XMLSchema-instance" xsd:noNamespaceSchemaLocation="http://www.web3d.org/specifications/x3d-3.0.xsd">
    <head>
        <meta name="filename" content="tmpsnz74v00.x3d" />
        <meta name="generator" content="Blender 2.78 (sub 5)" />
    </head>
    <Scene>
        ...
    </Scene>
</X3D>
```